build.gradle

```diff
@@ -1,11 +1,17 @@
 plugins {
     id 'org.jetbrains.kotlin.jvm' version '1.3.50'
     id 'maven-publish'
-    id "org.hibernate.build.maven-repo-auth" version "3.0.0"
+    id 'nu.studer.credentials' version '1.0.7'
```

Comment:
Any reason we're on this specific version instead of the latest?

Reply:
This one is used in did-auth-case and generally I don't use "latest" to avoid changes in parts your not working on which can potentially break something without having a clue where to look.

Reply:
Or do you mean I should try 2.1?

Reply:
Ah, I didn't mean to use the keyword latest but the latest release as in version 2.1.

Reply:
I've tried it, it's working fine so I'll used that. PR is updated now